src/RoleObjects/SingleLineTextBoxData.js

```diff
@@ -62,7 +62,12 @@ export default class SingleLineTextBoxData extends InputTagData {
    * @param {boolean} enable - true if autocomplete should be enabled, false otherwise
```

Comment:
Just a reminder that we'll want to update the doc comments for both the setter and getter to match our changes

Reply:
Yeah I can update those once the backward compatibility stuff is sorted out.